avocado/core/plugin_interfaces.py

```diff
@@ -424,6 +424,16 @@ async def update_requirement_cache(runtime_task, result):
         :type result: `avocado.core.teststatus.STATUSES`
         """
 
+    @abc.abstractmethod
+    def is_operational(self):
+        """Checks whether this spawner is operationally capable to perform.
```

Comment:
I am not sure I understand the expression "operationally capable to perform". Perform tests? How about "start tasks"? Or "whether this spawner is fully able to spawn tasks"?